ethcore/snapshot/src/watcher.rs

```diff
@@ -115,6 +115,9 @@ impl ChainNotify for Watcher {
 			// frequency of snapshots and ensures more snapshots are produced from similar points in
 			// the chain.
 			.filter(|num| num % self.period == 0 )
+			.inspect(|num| {
+				trace!(target: "snapshot_sync", "Candidate for taking a snapshot: #{}", num);
```

Comment:
how many candidates did you observe in practice?

Reply:
1